version.json

```diff
@@ -0,0 +1 @@
+{ "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json", "version": "2.3", "publicReleaseRefSpec": [ "^refs/heads/master$", "^refs/heads/v\\d+(?:\\.\\d+)?$" ], "cloudBuild": { "setVersionVariables": true, "buildNumber": { "enabled": true, "includeCommitId": { "when": "always" } } } }
```

Comment:
is this version 2.3 the next uid2-shared version number?

Reply:
2.3.1-SNAPSHOT will be the next version. It is 2.3.0-SNAPSHOT is the current version